templates/markup_field/research_experiment.timepoints.twig

```diff
@@ -0,0 +1,10 @@
+<div class="instructions years">
+  <p>We want to include information about  the experiment was performed. <strong>For multiple dates use the "Add More" button to add a form element for each one and do NOT use ranges.</strong></p>
```

Comment:
This first sentence looked incomplete. Here's my suggestion to keep consistent with Locations:
```suggestion
  <p>Timepoints refers to the time period the experiment was performed, which depending on the experiment, may cover one or multiple growing seasons, or a single day. <strong>For multiple dates, use the "Add More" button to add a form element for each one and do NOT use ranges.</strong></p>
```